From 5d843743c119c1d029a70be45067178e668d3097 Mon Sep 17 00:00:00 2001
From: Namit Solanki <nsolanki@codeaurora.org>
Date: Fri, 30 Sep 2016 15:54:39 +0530
Subject: [PATCH 40/43] surfaceflinger: Increase SurfaceFlinger dump buffer
 size to 16k.

 - Increase the SurfaceFlinger dump buffer size from 4k to 16k bytes to avoid buffer overflow.
---
 services/surfaceflinger/DisplayHardware/HWComposer_hwc1.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/services/surfaceflinger/DisplayHardware/HWComposer_hwc1.cpp b/services/surfaceflinger/DisplayHardware/HWComposer_hwc1.cpp
index dcb29135b..fb750ca38 100644
--- a/services/surfaceflinger/DisplayHardware/HWComposer_hwc1.cpp
+++ b/services/surfaceflinger/DisplayHardware/HWComposer_hwc1.cpp
@@ -1262,7 +1262,7 @@ void HWComposer::dump(String8& result) const {
     }
 
     if (mHwc && mHwc->dump) {
-        const size_t SIZE = 4096;
+        const size_t SIZE = 16*1024;
         char buffer[SIZE];
         mHwc->dump(mHwc, buffer, SIZE);
         result.append(buffer);
-- 
2.11.0

